global.json

```diff
@@ -1,6 +0,0 @@
-{
-  "sdk": {
-    "version": "3.1.100",
-    "rollForward": "latestMinor"
-  }
-}
```

Comment:
Maybe we should keep it? Or is OmniSharp still refusing to cooperate?

Reply:
Works fine now, so I'm bringing this back